Gems/Atom/Feature/Common/Assets/Shaders/Materials/Eye/Eye_SurfaceEval.azsli

```diff
@@ -35,23 +60,36 @@ Surface EvaluateSurface_Eye(
 
     real3x3 uvMatrix = CreateIdentity3x3(); // The eye material type doesn't support UV transforms
 
-    // Use a sigmoid to determine the sclera/iris contribution for each point
-    real distFromCenter = length(real2(localPosition.xz));
-    real mask = 1.0/(1.0 + exp(-(distFromCenter - real(MaterialSrg::m_eyeIrisRadius)) / (distFromCenter * real(MaterialSrg::m_limbusSize))));
+#if USE_NORMALIZED_SPACE_ALGORITHM
+    real3 normalizedLocal = normalize(localPosition);
+    real projection = dot(normalizedLocal, real3(0.0, 1.0, 0.0));
+    // If the localPosition of the vertex is at the center of the pupil, then the dot product
+    // with the forward vector will be at maximum value = 1.0.
+    // But the iris is in the XZ plane.
+    real tempDistFactor = sqrt(1 - projection * projection);
+    const real distFromCenter = tempDistFactor * NORMALIZED_EYE_RADIUS;
+#else
+    // This is the original equation used by the first developer of this shader.
+    // This equation suffers issues when the vertex data coming from the FBX changes
+    // from the original assumption that the radius of the eye mesh is 0.5.
+    const real distFromCenter = length(localPosition.xz);
+#endif
+
+    real mask = GetIrisRadiusMask(distFromCenter);
 
     // ------- Normal -------
-    
+
     surface.vertexNormal = normalize(vertexNormal);
-    
+
     // Get surface normal for each layer (iris/sclera) and blend using the mask defined above
-    float2 irisNormalUv = uvs[MaterialSrg::m_iris_m_normalMapUvIndex];
+    real2 irisNormalUv = uvs[MaterialSrg::m_iris_m_normalMapUvIndex];
```

Comment:
Are you aware you are switching UVs to half floats and hope that this is intentional? When I added half float support I made sure to not switch position and uv data in case it may cause issues.